src/transformers/utils/bitsandbytes.py

```diff
@@ -84,7 +84,7 @@ class `Int8Params` from `bitsandbytes`.
             module._parameters[tensor_name] = new_value
 
 
-def replace_8bit_linear(model, threshold=6.0, modules_to_not_convert="lm_head", current_key_name=None):
+def replace_8bit_linear(model, threshold=6.0, modules_to_not_convert=["lm_head"], current_key_name=None):
```

Comment:
Please avoid using mutables as default arguments in functions: replace the list with None and set within the function. 

```suggestion
def replace_8bit_linear(model, threshold=6.0, modules_to_not_convert=None, current_key_name=None):
```

and then below on L116: 

```
    modules_to_not_convert = ["lm_head"] if modules_to_not_convert is None else modules_to_not_convert
```